source/material_model/entropy_model.cc

```diff
@@ -347,6 +498,19 @@ namespace aspect
                              "cohesion value (1e20 Pa) prevents the viscous stress from "
                              "exceeding the yield stress. Units: \\si{\\pascal}.");
 
+          // Multicomponent equilibration parameters
+          prm.declare_entry ("Maximum iteration for multicomponent equilibration", "50",
+                             Patterns::Double (0.),
+                             "The maximum allowed number of iterations for multicomponent equlibrating "
+                             "to reach the tolerance value. If the maximum iteration is reached but the"
+                             "temperature has not been equilibrated, the model run will abort."
+                             "In this case, you may run it with debug mode for backtrace");
```

Comment:
I would remove the last line. This is a general advice that is true for all models that abort, not specific for this parameter.